### PR TITLE
Update with new options

### PR DESCRIPTION
**Pull request by 5axes**
As I have made the job for my own test .

But of course tyour are free to cancel this request

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "Arc Welder plugin",
     "author": "fieldOfView",
-    "version": "3.5.0-DEV",
+    "version": "3.5.1-DEV",
```
Comment (fieldOfView) [inline plugin.json:4]: Please revert this change. The version will remain 3.5.0-DEV until an official 3.5.0 version is published to the Marketplace.

File: bin/win64/help.txt [hunk 2]
```diff
@@ -0,0 +1,75 @@
+
```
Comment (fieldOfView) [inline bin/win64/help.txt:1]: Please remove this file.

File: README.md [hunk 3]
```diff
@@ -4,4 +4,6 @@ This plugin wraps [ArcWelderLib](https://github.com/FormerLurker/ArcWelderLib) b
 
 Inspired by on a postprocessing script by [yysh12](https://github.com/yysh12)
 
-For further informatio about ArcWelder antistutter, see https://plugins.octoprint.org/plugins/arc_welder
\ No newline at end of file
+For further informations about ArcWelder antistutter, see https://plugins.octoprint.org/plugins/arc_welder
```
Comment (fieldOfView) [inline README.md:7]: Thanks for ficing this typo, however `information` has on plural form in english. `For further information about ...`

File: ArcWelderPlugin.py [hunk 4]
```diff
@@ -112,6 +113,14 @@ def _filterGcode(self, output_device):
         maximum_radius = global_container_stack.getProperty("arcwelder_maximum_radius", "value")
         tolerance = global_container_stack.getProperty("arcwelder_tolerance", "value") / 100
         resolution = global_container_stack.getProperty("arcwelder_resolution", "value")
+        min_arc = int(global_container_stack.getProperty("arcwelder_min_arc_segment", "value"))
+        mm_par_arc = global_container_stack.getProperty("arcwelder_mm_per_arc_segment", "value")
```
Comment (fieldOfView) [inline ArcWelderPlugin.py]: Please change to `mm_per_arc_segment` (per, not par)

File: ArcWelderPlugin.py [hunk 5]
```diff
@@ -112,6 +113,14 @@ def _filterGcode(self, output_device):
         maximum_radius = global_container_stack.getProperty("arcwelder_maximum_radius", "value")
         tolerance = global_container_stack.getProperty("arcwelder_tolerance", "value") / 100
         resolution = global_container_stack.getProperty("arcwelder_resolution", "value")
+        min_arc = int(global_container_stack.getProperty("arcwelder_min_arc_segment", "value"))
```
Comment (fieldOfView) [inline ArcWelderPlugin.py]: Please change to `min_arc_segment`

File: ArcWelderPlugin.py [hunk 6]
```diff
@@ -112,6 +113,14 @@ def _filterGcode(self, output_device):
...
+        mm_par_arc = global_container_stack.getProperty("arcwelder_mm_per_arc_segment", "value")
+
+        
+        cmd_3d_arcs = ""
+        allow_3d_arcs = global_container_stack.getProperty("arcwelder_allow_3d_arcs", "value")
+        if allow_3d_arcs:
+            cmd_3d_arcs = "-z"
```
Comment (fieldOfView) [inline ArcWelderPlugin.py]: This could be written as 
`cmd_3d_arcs = "-z" if global_container_stack.getProperty("arcwelder_allow_3d_arcs", "value") else ""`

File: ArcWelderPlugin.py [hunk 7]
```diff
@@ -134,8 +143,9 @@ def _filterGcode(self, output_device):
                 with os.fdopen(file_descriptor, 'w') as temporary_file:
                     temporary_file.write(joined_gcode)
 
-                subprocess.run([arcwelder_path, "-m=%f" % maximum_radius, "-t=%f" % tolerance, "-r=%f" % resolution, path])
-
+                # Logger.log("d", "Process.run %s , %s, %s, %s, %s, %s, %s, %s", arcwelder_path, cmd_3d_arcs, "-m=%f" % maximum_radius, "-a=%d" % min_arc, "-s=%f" % mm_par_arc, "-t=%f" % tolerance, "-r=%f" % resolution, path)
```
Comment (fieldOfView) [inline ArcWelderPlugin.py]: Please remove the commented-out line.

File: README.md [hunk 8]
```diff
@@ -4,4 +4,6 @@ This plugin wraps [ArcWelderLib](https://github.com/FormerLurker/ArcWelderLib) b
...
+
+For further informations about MM Per Arc Segment and Min Arc Segments options, see https://github.com/FormerLurker/ArcWelderPlugin/issues/80
```
Comment (fieldOfView) [inline README.md:9]: I'm not sure if we should link to specific issues for specific features. Ideally, the description of the setting would be good enough, and eventually I expect any documentation in that issue to trickle down to the official documentation of the arcwelder plugin.

File: ArcWelderPlugin.py [hunk 9]
```diff
@@ -1,5 +1,6 @@
 # Copyright (c) 2020 Aldo Hoeben / fieldOfView
 # The ArcWelderPlugin for Cura is released under the terms of the AGPLv3 or higher.
+# 25/11/2020 add options arcwelder_min_arc_segment / arcwelder_mm_per_arc_segment / arcwelder_allow_3d_arcs
```
Comment (fieldOfView) [inline ArcWelderPlugin.py:3]: I don't keep a changelog inside my plugins.

**Review by fieldOfView — CHANGES_REQUESTED** [review]
Thanks for the PR. I have some suggested changes.

**Comment by 5axes** [comment]
Ok finally too many differences between my code and what you would like to have. So I close the pull request.

**Comment by fieldOfView** [comment]
Hey @5axes, this is how pull requests work; I could also make these changes after merging in your version, but then you lay the burden with me. I don't mean to micromanage you, and appreciate your contribution, but the end-goal is to have a consistent codebase, with a consistent code style.
